pkg/controller/job/job_controller_test.go

```diff
@@ -1018,24 +910,6 @@ func TestGetStatus(t *testing.T) {
 		wantSucceeded        int32
 		wantFailed           int32
 	}{
-		"without finalizers": {
-			job: batch.Job{
-				Status: batch.JobStatus{
-					Succeeded: 1,
-					Failed:    2,
-				},
-			},
-			pods: []*v1.Pod{
-				buildPod().uid("a").phase(v1.PodSucceeded).Pod,
-				buildPod().uid("b").phase(v1.PodSucceeded).Pod,
-				buildPod().uid("c").phase(v1.PodFailed).Pod,
-				buildPod().uid("d").phase(v1.PodFailed).Pod,
-				buildPod().uid("e").phase(v1.PodFailed).Pod,
-				buildPod().uid("f").phase(v1.PodRunning).Pod,
-			},
-			wantSucceeded: 2,
-			wantFailed:    3,
-		},
 		"some counted": {
 			job: batch.Job{
 				Status: batch.JobStatus{
```

Comment:
Remove `UncountedTerminatedPods` from the test cases, as whether that's nil should be ignored.

Reply:
Done. I think you meant to remove the check of `UncountedTerminatedPods` being nil.

Reply:
Oh that too.

But I mean lines like this shouldn't be needed anymore.

```golang
UncountedTerminatedPods: &batch.UncountedTerminatedPods{}
```